modules/gapi/src/streaming/onevpl/engine/preproc/preproc_engine.cpp

```diff
@@ -34,6 +34,36 @@ bool FrameInfoComparator::equal_to(const mfxFrameInfo& lhs, const mfxFrameInfo&
     return lhs == rhs;
 }
 
+cv::util::optional<cv::Rect> apply_roi(mfxFrameSurface1* surface_handle,
+                                       cv::util::optional<cv::Rect> &&opt_roi,
+                                       const char *preamb = "apply ROI: ") {
```

Comment:
y do we need this `preamb` here? Isn't the `__FUNCTION__` thing included in `_LOG_DEBUG`?